packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -0,0 +1,188 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { BoxelInput } from '@cardstack/boxel-ui';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { trackedFunction } from 'ember-resources/util/function';
+import type { CardDef } from 'https://cardstack.com/base/card-api';
+import type CardService from '../../services/card-service';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    url: URL;
+    onEnterPressed: (url: URL) => void;
+    card: CardDef | null;
+    cardError: string | null;
+    resetCardError: () => void;
+  };
+}
+
+export default class CardURLBar extends Component<Signature> {
+  <template>
+    <div
+      id='card-url-bar'
+      class={{this.cssClasses}}
+      data-test-card-url-bar
+      ...attributes
+    >
+      <div class='realm-info' data-test-card-url-bar-realm-info>
+        <img src={{this.realmIcon}} alt='realm-icon' />
+        <span>in
+          {{if this.realmName this.realmName 'Unknown Workspace'}}</span>
+      </div>
+      <div class='input'>
+        {{svgJar 'icon-globe' width='22px' height='22px'}}
+        <BoxelInput
+          class='url-input'
+          @value={{this.url}}
+          @onInput={{this.onInput}}
+          @onKeyPress={{this.onKeyPress}}
+          @onFocus={{this.toggleFocus}}
+          @onBlur={{this.toggleFocus}}
+          data-test-card-url-bar-input
+        />
+      </div>
+      {{#if this.showErrorMessage}}
+        <div class='error-message' data-test-card-url-bar-error>
+          <span>{{this.errorMessage}}</span>
+        </div>
+      {{/if}}
+    </div>
+    <style>
+      .card-url-bar {
+        display: flex;
+        align-items: center;
+
+        background: var(--boxel-purple-700);
```

Comment:
it's sufficient to just use `background-color` property when that's the only thing you're changing